packages/core/strapi/lib/Strapi.js

```diff
@@ -570,10 +579,15 @@ class Strapi {
   }
 }
 
-module.exports = (options) => {
-  const strapi = new Strapi(options);
+let strapi;
+
+const initialize = (config = {}) => {
+  strapi = new Strapi(config);
```

Comment:
What happens if a Strapi instance is already initialized? Should we destroy it or let it go?
Right now we're assuming someone else will cleanup but we have the opportunity to let the init cleanup after itself thanks to the singleton

Reply:
we could call the destroy good call :) I will do that. At the moment it was not necessary as there is no place where we initialize two strapi apps (I think) but that could be a great addition